### PR TITLE
Unblind mll

### PR DESCRIPTION
**Pull request by kdlong**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -594,19 +594,19 @@ jobs:
...
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/plotting/makeDataMCStackPlot.py --yscale 1.3 --baseName nominal --nominalRef nominal --hists etaAbsEta-mll mll -o $WEB_DIR -f $PLOT_DIR -p z $HIST_FILE variation --varName 'virtual_ewCorr' --selectEntries 2 --varLabel 'EW(virtual)' --selectAxis systIdx --color blue 
 
       - name: dilepton combine mll setup
-        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/combine/setupCombine.py -i $HIST_FILE --fitvar etaAbsEta-mll-cosThetaStarll --rebin 1 1 10 --fitMassDiff cosThetaStarll --lumiScale $LUMI_SCALE -o $WREMNANTS_OUTDIR
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/combine/setupCombine.py -i $HIST_FILE --fitvar etaAbsEta-mll cosThetaStarll --lumiScale $LUMI_SCALE -o $WREMNANTS_OUTDIR --realData
```
Comment (bendavid) [inline .github/workflows/main.yml:603]: Why does cosThetaStarll still appear here at all?

Comment (bendavid) [reply .github/workflows/main.yml:603]: Ah I guess as a second fit var it anyways gets ignored since there was only one input file?

Comment (bendavid) [reply .github/workflows/main.yml:603]: Yes ok this can be cleaned up after.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -594,19 +594,19 @@ jobs:
...
 
       - name: dilepton combine mll fit
-        run: cmssw-cc7 --command-to-run scripts/ci/run_combine.sh /home/c/cmsmwbot/combinetf/CMSSW_10_6_30/src/ mass $WREMNANTS_OUTDIR/ZMassDilepton_etaAbsEta_mll_cosThetaStarll ZMassDilepton_inclusive.txt 
+        run: cmssw-cc7 --command-to-run scripts/ci/run_combine.sh /home/c/cmsmwbot/combinetf/CMSSW_10_6_30/src/ mass $WREMNANTS_OUTDIR/ZMassDilepton_etaAbsEta_mll ZMassDilepton_inclusive.txt
```
Comment (bendavid) [inline .github/workflows/main.yml:606]: I don't think this actually unblinds, since it still runs the fit with -t -1

**Comment by bendavid** [comment]
The ci needs to still be cleaned up a bit, but I'll merge this so that we can tag it and start working through the other PR's.

**Comment by bendavid** [comment]
From the rest of the ci the expected eff syst uncertainty on mW visibly decreased, even with the reduced stats, so that's as expected at least.

**Comment by bendavid** [comment]
The other ambiguity we should still solve is hdf5 vs root output for setupCombine (or be REALLY sure they give the same result)

File: .github/workflows/main.yml (hunk 1 above)
Comment (kdlong) [reply .github/workflows/main.yml:603]: Oops accident, yes, would be better to fix this.

File: .github/workflows/main.yml (hunk 2 above)
Comment (kdlong) [reply .github/workflows/main.yml:606]: Ah fuck, -1 is inside run_combine.sh?

Comment (bendavid) [reply .github/workflows/main.yml:606]: Yes

**Comment by kdlong** [comment]
In #454 I made hdf5 the default (sorry forgot to point that out, but it was annoying me). Don't we want to do that?

File: .github/workflows/main.yml (hunk 2 above)
Comment (bendavid) [inline .github/workflows/main.yml:606]: Isn't this command explicitly referencing the text datacard output rather than the hdf5?

**Comment by bendavid** [comment]
Yes I think we DO want hdf5 as default, but it looks to me that the ci is still using the text/root output in some places.

**Comment by kdlong** [comment]
Yes, it does. I think this was intentional since the root output isn't totally deprecated, so this at least validates that it works

**Pull request merged**